### DISCUSSION TITLE
Prompt before changing pattern

### DISCUSSION THREAD

**Discussion by bjorn-felle** [bangcorrupt/hacktribe#130] [Q&A] [answered]
The pattern change knob doubling as a data entry knob makes it a bit too easy to change pattern when you didn't mean to. The lack of confirmation makes it possible to easily lose work this way. Is there any possibility of adding a confirmation to changing patterns if there are unsaved changes?

**Comment by maks** [comment, marked as answer]
I guess you are asking for an alternative mechanism to the existing one where you can change the global setting to require shift+dial to switch between patterns?

**Reply by weicj** [reply]
Yep, that function called "Ptn Change Lock" on the "Global Parameter", simply set it to "ON", and your problem would be solved (like maks says "shit+dial" to change pattern). Just can't understand why Korg did not turn it "ON" by default, and you cannot even find such information on the manual.

**Comment by bjorn-felle** [comment]
Thank you, I didn't know about that option. Don't talk to me about Korg manuals xD thanks folks!
